politicoAPI/app/api/v1/views/politicalparty.py

```diff
@@ -0,0 +1,36 @@
+from flask import request, Blueprint, jsonify, make_response
+import json
+from app.api.v1.models.political_model import Political
+
+politicalparty= Blueprint('politicalparty',__name__,url_prefix='/api/politicalparty')
+
+class Party():
+    @politicalparty.route('/parties',methods=['POST'])
+    def create_political_party():
+        party = request.get_json()
+        name = party['name']
+        hqAddress = party['hqAddress']
+        logoUrl = party['logoUrl']
+
+        feedback = Political().create_political_party('id', 'name', 'hqAddress', 'logoUrl')
```

Comment:
What data are you passing through? is it variables or hardcoded ones

Reply:
i rectified that part